### PR TITLE
Inspect transformers

### LINKED ISSUE #22712: Inspect: Transformation in data tab

### PR DESCRIPTION
**Pull request by torkelo**
Closes https://github.com/grafana/grafana/issues/22712
Continuation of this work, had to copy the code as the refactoring done in another PR was making continuing on that PR impossible. 
https://github.com/grafana/grafana/pull/23295

* [x] Basic UX 
* [x] Apply transformations 

Screenshot from 2020-04-15 18-34-36

### CODE REVIEW

File: public/app/features/dashboard/components/Inspector/InspectDataTab.tsx [hunk 1]
```diff
@@ -28,8 +46,44 @@ export class InspectDataTab extends PureComponent<Props> {
...
+    if (currentTransform && currentTransform.transformer.id !== DataTransformerID.noop) {
+      return transformDataFrame([currentTransform.transformer], data);
+    }
+    return data;
+  }
+
+  getProcessedData(): DataFrame[] {
+    return applyFieldOverrides({
+      data: this.getTransformedData(),
+      theme: config.theme,
```
Comment (hugohaggmark) [inline public/app/features/dashboard/components/Inspector/InspectDataTab.tsx:76]: Out of the scope for this PR but passing theme to field overrides seems weird to me.

Comment (torkelo) [reply public/app/features/dashboard/components/Inspector/InspectDataTab.tsx:76]: It's used to derive threshold colors from the palette

Comment (dprokop) [reply public/app/features/dashboard/components/Inspector/InspectDataTab.tsx:76]: I have a similar problem with this, but haven't found a better way:(

**Review by hugohaggmark — CHANGES_REQUESTED** [review]
Looks good but I managed to crash Grafana:
image

JSON:
{
  "datasource": null,
  "aliasColors": {},
  "bars": false,
  "dashLength": 10,
  "dashes": false,
  "fieldConfig": {
    "defaults": {
      "custom": {
        "align": null
      },
      "thresholds": {
        "mode": "absolute",
        "steps": [
          {
            "color": "green",
            "value": null
          },
          {
            "color": "red",
            "value": 80
          }
        ]
      },
      "mappings": []
    },
    "overrides": []
  },
  "fill": 1,
  "fillGradient": 0,
  "gridPos": {
    "h": 9,
    "w": 12,
    "x": 0,
    "y": 0
  },
  "hiddenSeries": false,
  "id": 2,
  "legend": {
    "avg": false,
    "current": false,
    "max": false,
    "min": false,
    "show": true,
    "total": false,
    "values": false
  },
  "lines": true,
  "linewidth": 1,
  "nullPointMode": "null",
  "options": {
    "showHeader": true,
    "resizable": false,
    "angular": {
      "aliasColors": {},
      "bars": false,
      "dashLength": 10,
      "dashes": false,
      "fill": 1,
      "fillGradient": 0,
      "hiddenSeries": false,
      "legend": {
        "avg": false,
        "current": false,
        "max": false,
        "min": false,
        "show": true,
        "total": false,
        "values": false
      },
      "lines": true,
      "linewidth": 1,
      "nullPointMode": "null",
      "options": {
        "dataLinks": []
      },
      "percentage": false,
      "pointradius": 2,
      "points": false,
      "renderer": "flot",
      "seriesOverrides": [],
      "spaceLength": 10,
      "stack": false,
      "steppedLine": false,
      "thresholds": [],
      "timeRegions": [],
      "tooltip": {
        "shared": true,
        "sort": 0,
        "value_type": "individual"
      },
      "xaxis": {
        "buckets": null,
        "mode": "time",
        "name": null,
        "show": true,
        "values": []
      },
      "yaxes": [
        {
          "$$hashKey": "object:320",
          "format": "short",
          "label": null,
          "logBase": 1,
          "max": null,
          "min": null,
          "show": true
        },
        {
          "$$hashKey": "object:321",
          "format": "short",
          "label": null,
          "logBase": 1,
          "max": null,
          "min": null,
          "show": true
        }
      ],
      "yaxis": {
        "align": false,
        "alignLevel": null
      }
    }
  },
  "percentage": false,
  "pluginVersion": "7.0.0-pre",
  "pointradius": 2,
  "points": false,
  "renderer": "flot",
  "seriesOverrides": [],
  "spaceLength": 10,
  "stack": false,
  "steppedLine": false,
  "targets": [
    {
      "alias": "A",
      "refId": "A",
      "scenarioId": "table_static",
      "stringInput": ""
    },
    {
      "alias": "B",
      "refId": "B",
      "scenarioId": "table_static",
      "stringInput": ""
    }
  ],
  "thresholds": [],
  "timeFrom": null,
  "timeRegions": [],
  "timeShift": null,
  "title": "Panel Title",
  "tooltip": {
    "shared": true,
    "sort": 0,
    "value_type": "individual"
  },
  "type": "table",
  "xaxis": {
    "buckets": null,
    "mode": "time",
    "name": null,
    "show": true,
    "values": []
  },
  "yaxes": [
    {
      "$$hashKey": "object:320",
      "format": "short",
      "label": null,
      "logBase": 1,
      "max": null,
      "min": null,
      "show": true
    },
    {
      "$$hashKey": "object:321",
      "format": "short",
      "label": null,
      "logBase": 1,
      "max": null,
      "min": null,
      "show": true
    }
  ],
  "yaxis": {
    "align": false,
    "alignLevel": null
  }
}

crash-transformer

**Comment by torkelo** [comment]
Probably the dataFrameIndex that needs checks that is not above length. If anyone could help fix that it would be great

**Comment by hugohaggmark** [comment]
> Probably the dataFrameIndex that needs checks that is not above length. If anyone could help fix that it would be great

I'll look into it

**Review by hugohaggmark — APPROVED** [review]
LGTM!

**Pull request merged**